### ISSUE TITLE
Can't validate numeric, number values

### ISSUE THREAD

**Issue by insamo** [go-playground/validator#356]
### Package version eg. v8, v9: 

v9

### Issue, Question or Enhancement:

Hello, I can't validate integers varaibles, maybe i do something wrong

go version go1.10.1 linux/amd64

### Code sample, to showcase or reproduce:

```go
package main

import (
	"fmt"
	"gopkg.in/go-playground/validator.v9"
)

func main() {
	var dd int64
	dd = 3000000
	validate := validator.New()
	if errs := validate.Var(dd, "required,numeric"); errs != nil {
		fmt.Println(errs)
	}

	if errs := validate.Var(dd, "required,number"); errs != nil {
		fmt.Println(errs)
	}
}

```
Returns
```
Key: '' Error:Field validation for '' failed on the 'numeric' tag
Key: '' Error:Field validation for '' failed on the 'number' tag
```
Thanks

**Comment by joeybloggs** [comment]
I'll try and take a look soon @insamo

**Comment by insamo** [comment]
Thx @joeybloggs

**Comment by joeybloggs** [comment]
@insamo corrected this behaviour in [v9.17.0](https://github.com/go-playground/validator/releases/tag/v9.17.0)

this was originally only intended to be used on a string but makes sense when dealing with `interface{}`
